### DISCUSSION TITLE
Startup into file - be as fast as vanilla nvim?

### DISCUSSION THREAD

**Discussion by thyttan** [LazyVim/LazyVim#212] [Q&A] [answered]
Hi!

My personal config (using Lazy.nvim) launches practically immediately having everything accessible and visible at once, whereas LazyVim loads just that second or so later in comparison.

After @folke's instruction I've looked at the lazy profile tab and I get these startup times:

30-50 ms over 5 tests launching directly to the LazyVim staring screen.

150-170 ms when opening a 250 line js-file.

But it feels like it takes around 700 ms before the lines are visible (closer to 1000 ms before statusline shows up)

Video for reference (opening a new file "test"): [Screencast from 2023-02-06 18-46-37.webm](https://user-images.githubusercontent.com/97237430/217528552-68d04cb0-296d-4d0d-b96f-16f4a58ae4d9.webm)

Is there a way to have the lines be 'instantly' visible when launching into a file with lazyvim as with my previous config?

This is of course a bit nitpicky, but it would be awesome if there was something I could do. Maybe disabling some of the (very ;) ) fancy UI?

Thanks again and in advance!

**Comment by folke** [comment, marked as answer]
That's odd. What OS and terminal are you using? Can you do a full nvim --startuptime? I've heard people having issues with the Neovim clipboard functionality. Definitely not lazyvim or lazy.nvim, since you can that 30ms you see is after everything loaded. 

Lua line is loaded on very lazy though, so maybe you have other plugins also loading on that event that take a long time?

Can you post the full profile text from lazy profile? It should show if another plugin is taking a lot of time.

For me lazyvim opens instantly. About 16ms

**Reply by thyttan** [reply]
Thank you for the swift reply once more :)

>What OS and terminal are you using?

Fedora 37, gnome-terminal with fish-shell as login shell.

Fish shell is not standard so maybe that's the culprit? It doesn't seem entirely likely to me, I feel like it shouldn't have to load more than once at launch of the terminal? See screen capture: 
[Screencast from 2023-02-08 14-45-01.webm](https://user-images.githubusercontent.com/97237430/217547343-52d82380-2663-4283-8ac9-0f0a45e6e078.webm)


>Can you do a full nvim --startuptime?

[nvim-startuptime-thyttan.txt](https://github.com/LazyVim/LazyVim/files/10686002/nvim-startuptime-thyttan.txt)

>Lua line is loaded on very lazy though, so maybe you have other plugins also loading on that event that take a long time?

I think I noticed the behavior before doing any changes with regards to plugins or options. I just removed the stuff I added and tried again to make sure, still get the same result.

>Can you post the full profile text from lazy profile? It should show if another plugin is taking a lot of time.

[lazyvim-profile-tab-thyttan.txt](https://github.com/LazyVim/LazyVim/files/10686028/lazyvim-profile-tab-thyttan.txt)

**Reply by folke** [reply]
Check your startuptime log. clipboard is indeed the issue. It takes over 700ms to load before thr buffer will be displayed.

**Reply by thyttan** [reply]
Thank you! 

I will look into this then:
> I've heard people having issues with the Neovim clipboard functionality.
https://github.com/LazyVim/LazyVim/discussions/147

**Reply by thyttan** [reply]
Now solved, thanks!

>I had this problem on Fedora 37, gnome terminal. Fixed by installing `wl-clipboard` where `wl-copy` lives. Thanks!

_Originally posted by @thyttan in https://github.com/LazyVim/LazyVim/discussions/147#discussioncomment-4905648_
